src/Nethermind/Nethermind.Merge.Plugin/Synchronization/BeaconPivot.cs

```diff
@@ -56,14 +56,14 @@ public class BeaconPivot : IBeaconPivot
 
         public long PivotNumber => CurrentBeaconPivot?.Number ?? _syncConfig.PivotNumberParsed;
 
-        public Keccak PivotHash => CurrentBeaconPivot?.Hash ?? _syncConfig.PivotHashParsed;
+        public Keccak PivotHash => CurrentBeaconPivot?.Hash ?? _syncConfig.PivotHashParsed!;
```

Comment:
What if it is actually null? Is this a problem?

Reply:
I have the same concerns